app/api/localStorage/primitives.js

```diff
@@ -0,0 +1,157 @@
+// @flow
+
+import environment from '../../environment';
+
+/**
+ * The `chrome` variable refers to the JavaScript APIs for WebExtensions that can be accessed
+ * by Firefox and Chrome. As its name says, this can be accessed only by the WebExtension.
+ * so the WebPage will use the `localStorage`.
+ */
+
+declare var chrome;
+
+// =====
+//  get
+// =====
+
+const getStorageItemInExtension = async (
+  key: string | void
+): Promise<?string> => new Promise((resolve, reject) => {
+  chrome.storage.local.get(key, (data: {}, error) => {
```

Comment:
@SebastienGllmt 

https://developer.chrome.com/extensions/storage#method-StorageArea-get
According to `chrome.storage` specification it seems out `error` parameter will not be set instead global `runtime.lastError`  will be set.
![image](https://user-images.githubusercontent.com/19986226/63486881-36a6cf00-c4e4-11e9-9088-d6269ed43934.png)


Looked at some examples:
https://stackoverflow.com/questions/15143439/how-to-check-if-a-variable-is-set-on-chrome-storage?answertab=votes#tab-top

https://dev.to/milandhar/chrome-local-storage-in-extensions-4k9m